test/fixtures/served/ampdoc-with-messaging.html

```diff
@@ -21,7 +21,10 @@
   <link rel="canonical" href="amps.html" >
   <meta name="viewport" content="width=device-width,minimum-scale=1">
   <script async src="../../../dist/amp.js"></script>
+  <!-- This line makes the tests pass on Travis. -->
   <script async src="../../../dist/v0/amp-viewer-integration-0.1.js"></script>
```

Comment:
Not sure why that would happen. If we already have `amp.js` on top, that means that we have `max` versions in both places. No?